deploy/efs-csi/01-daemonset.yaml

```diff
@@ -0,0 +1,99 @@
+# Source: https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/51d19a433dcfc47fbb7b7a0e1c8ff6ab98ce87e9/deploy/kubernetes/base/node.yaml
```

Comment:
Can you add a YAML document separator `---` at the top of all of these manifests?

Reply:
I was going to, but am following precedent. Only four out of the existing 133 yaml files under `deploy` do so. The builder is putting the chunks into a single yaml file anyway, so arguably sans-separator makes more sense. :shrug: